packages/block-library/src/image/edit.native.js

```diff
@@ -61,6 +61,7 @@ class ImageEdit extends React.Component {
 		this.onSelectMediaUploadOption = this.onSelectMediaUploadOption.bind( this );
 		this.updateMediaProgress = this.updateMediaProgress.bind( this );
 		this.updateAlt = this.updateAlt.bind( this );
+		this.updateSlider = this.updateSlider.bind( this );
```

Comment:
changes in this file will be removed before PR to WordPress repo (it's just to allow see how RangeCell works)